dkpro-core-opennlp-asl/src/test/java/de/tudarmstadt/ukp/dkpro/core/opennlp/OpenNlpPosTaggerTest.java

```diff
@@ -145,7 +145,7 @@ public void testEnglishManualURI()
     public void testAfrikaans()
         throws Exception
     {
-        runTest("af", "afribooms", "Betalings aan die SAID kan op enigeen van die volgende maniere gemaak word .",
+        runTest("af", "afribooms-perceptron", "Betalings aan die SAID kan op enigeen van die volgende maniere gemaak word .",
```

Comment:
How about adding the new model as a default for `af` in `dkpro-core-opennlp-asl/src/main/resources/de/tudarmstadt/ukp/dkpro/core/opennlp/lib/tagger-default-variants.map`?